needy/__main__.py

```diff
@@ -184,6 +184,10 @@ def main(args=sys.argv):
 
     lock_fd = os.open('.needy_lock', os.O_RDWR | os.O_CREAT)
     try:
+        try:
+            fcntl.flock(lock_fd, fcntl.LOCK_EX | fcntl.LOCK_NB)
+        except IOError:
+            print('Unable to lock .needy_lock, please be sure that only one instance of needy is running')
```

Comment:
i think the message should be a little less... erroneous sounding. it's definitely not an issue to have more than one instance. the user just may need to be aware of what's happening. so how about something a little nicer like "waiting for other instances to terminate..."?

mention of .needy_lock should probably be avoided unless we're running verbosely (which is another issue)